assignments/objects.js

```diff
@@ -18,47 +18,109 @@ const example = {
 }
 
 // Write your intern objects here:
-
-
-// ==== Challenge 2: Reading Object Data ==== 
+const interns = [
+  {
+    "id": 1,
+    "name": "Mitzi",
+    "email": "mmelloy0@psu.edu",
+    "gender": "F"
+ },
+ {
+    "id": 2,
+    "name": "Kennan",
+    "email": "kdiben1@tinypic.com",
+    "gender": "M",
+    Speak: () => {
```

Comment:
I love arrow functions. Have you considered the difference between an arrow function and a regular function? Think about this for the future: https://medium.com/@thejasonfile/es5-functions-vs-es6-fat-arrow-functions-864033baa1a